packages/bot-web-ui/src/components/journal.jsx

```diff
@@ -105,78 +97,114 @@ const Tools = ({ checked_filters, filters, filterMessage }) => (
     </div>
 );
 
-const Journal = ({ filtered_messages, ...props }) => {
+const getJournalItemContent = (message, type, className, extra) => {
+    switch (type) {
+        case message_types.SUCCESS: {
+            return <FormatMessage logType={message} extra={extra} className={className} />;
+        }
+        case message_types.NOTIFY: {
+            return <div className={classnames('journal__text', className)}>{message}</div>;
+        }
+        case message_types.ERROR: {
+            return <div className='journal__text--error journal__text'>{message}</div>;
+        }
+        case message_types.COMPONENT: {
+            return <div className={classnames('journal__text-row', className)}>{message}</div>;
+        }
+        default:
+            return <></>;
```

Comment:
Same here, let's return `null` when nothing needs to be rendered. It's more consistent with the rest of the codebase.